src/commands/init.ts

```diff
@@ -113,8 +118,28 @@ configuration can also be overridden using environment variables.`
       {type: 'mask'})
     const clientSecret = await CliUx.ux.prompt('Your client secret?',
       {type: 'hide'})
+    const suggestedSystem = Config.getSystemFromApiServer(apiServer)
+    const system = await CliUx.ux.prompt('Mix system name?',
+      {default: suggestedSystem})
+
+    const values = {
+      authServer,
+      apiServer,
+      scope,
+      tenant,
+      clientId,
+      clientSecret,
+    }
 
-    const mixCLIConfig = {authServer, apiServer, scope, tenant, clientId, clientSecret}
+    const mixCLIConfig = {
+      ...values,
+      currentSystem: system.toLowerCase(),
```

Comment:
We should not update `currentSystem` in the config at the end of the `init` command. Here's the scenario where that fails:

- You are using the "us" Mix system and you are authenticated so the token file contains a vaild token for "us" for another 15 minutes.
- You run the `init` command to add the "eu" Mix system.
- mix-cli currently switches the current system to "eu".
- You run a regular command like "organizations:list".
- mix-cli hits the "eu" Mix system with what it thinks is a valid token.
- You get an error EUNAUTHORIZED because the token, while not expired, belongs to the "us" Mix system and not to the "eu" Mix system.

We tell users to use the `auth` command to switch. So let's stay consistent with that, not switch after `init` and the problem will be addressed.

This also brings back something we discussed yesterday: we should capture the system to which the token belongs to in the token file. I haven't checked if that was done. Let's discuss this morning.

Reply:
If we only switch the system with the `auth` command(which is also the one updating the token), why does the access_token file require a `currentSystem` property? Shouldn't the `currentSystem` in the config serve that purpose?

Reply:
As discussed, with the fix discussed above, we think we cover all the edge cases, so checking for `currentSystem` when using the token would be superfluous. We wont' do it at this time.